### PR TITLE
CSES Maximum Sliding Window II Solution + Explanation

### PR DESCRIPTION
**Pull request by nwatx**

### CODE REVIEW

**Review by thecodingwizard — CHANGES_REQUESTED** [review]
looks good, but can you make the code compile by either adding the necessary macros, or (preferably) undoing the macros? (and add bits/stdc++.h)

Thanks!

**Comment by nwatx** [comment]
should be good now 👍

**Comment by thecodingwizard** [comment]
.gitignore was accidentally deleted?

**Comment by nwatx** [comment]
fixed

File: solutions/CSES Maximum Subarray II.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+<Spoiler title="Hint 1">
+
+We can observe that we will be performing sum range queries for our subarray.
+
+Therefore, we should construct a prefix sum array to perform these queries.
+
+</Spoiler>
+
+# Solution
```
Comment (thecodingwizard) [inline solutions/CSES Maximum Subarray II.mdx:19]: if the hint is wrapped in a spoiler tag, then shouldn't the solution be hidden by a spoiler too?

**Comment by nwatx** [comment]
added solution tag

**Pull request merged**